packages/twenty-server/src/engine/metadata-modules/serverless-function/dtos/serverless-function.dto.ts

```diff
@@ -56,16 +57,15 @@ export class ServerlessFunctionDTO {
   @Field()
   sourceCodeHash: string;
 
-  @IsString()
-  @IsNotEmpty()
-  @Field()
-  sourceCodeFullPath: string;
-
   @IsString()
   @IsNotEmpty()
   @Field()
   runtime: string;
 
+  @IsNumber()
+  @Field({ nullable: true })
+  latestVersion: string;
```

Comment:
lastestPublishedVersion?

Reply:
And why use string over number?

Reply:
because aws lambda service returns strings, but indeed, I need to remove the IsNumber()

Reply:
lastestPublishedVersion -> not sure, it might also be `latestDeployedVersion`, I would say it is clear enough for now. What do you think?